content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,173 @@
+=================
+FedEx integration
+=================
+
+Integrating a FedEx account with Odoo's **Inventory** app makes it possible to :doc:`calculate
+shipping rates <delivery_method>`, and :doc:`generate shipping labels <labels>` within Odoo. This is
+accomplished by enabling the FedEx *shipping connector*, and then configuring at least one *shipping
+method*.
+
+.. note::
+   This documentation contains configuration details specific to FedEx integration. See the
+   documentation on :doc:`third-party shippers <third_party_shipper>` for general shipper
+   integration instructions.
+
+Enable shipping connector
+=========================
+
+To enable the shipping connector for FedEx, navigate to :menuselection:`Inventory app -->
+Configuration --> Settings`. Scroll down to the :guilabel:`Shipping Connectors` heading, and tick
+the checkbox next to :guilabel:`FedEx Connector`.
+
+Finally, click :guilabel:`Save` to save the changes. After doing so, a :icon:`oi-arrow-right`
+:guilabel:`FedEx Shipping Methods` button appears below :guilabel:`FedEx Connector`.
+
+.. image:: fedex/fsm-button.png
+   :align: center
+   :alt: The FedEx Shipping Methods button below the FedEx Connector.
+
+Configure shipping method
+=========================
+
+Once the FedEx shipping connector is enabled, it is necessary to configure at least one shipping
+method. After doing so, the shipping method can be included in sales orders (SOs), and used to
+compute shipping costs, and print shipping labels.
+
+To enable a shipping method, navigate to :guilabel:`Inventory app --> Configuration --> Settings`,
+and click the :guilabel:`FedEx Shipping Methods` button below the :guilabel:`FedEx Connector`
+checkbox. Doing so opens a page that shows all existing FedEx shipping methods.
+
+.. note::
+   To see all shipping methods for every shipper with a connector enabled, navigate to
+   :menuselection:`Inventory app --> Configuration --> Shipping Methods`.
+
+Select a shipping method to open its form. Alternatively, click :guilabel:`New` to open a blank
+form, and configure a new shipping method.
+
+.. image:: fedex/fedex-form.png
+   :align: center
+   :alt: The form for a FedEx shipping method.
+
+.. important::
+   Enabling the FedEx shipping connector automatically creates two default shipping methods: *FedEx
+   US* and *FedEx International*. Each of these methods are pre-configured with test credentials,
+   allowing them to be used for testing purposes.
+
+   Before the shipping method can be used to create actual shipments, the test credentials must be
+   replaced with credentials from a valid FedEx account.
+
+General information
```

Comment:
It might be nice to initially let the reader know that these forms are separated into general info, and three tabs...then you can have ref links embedded in that statement, so they can just quickly click to those areas, should they be so inclined. Just something to consider during your revisions. Thanks! 🫡